### ISSUE TITLE
`sauce --shell bash` pretends to source nonexistent file

### ISSUE THREAD

**Issue by ngirard** [DanCardin/sauce#28]
Using precompiled binary v0.6.5 on Ubuntu 20.04.

Observed behaviour:
```sh
sauce --shell bash
Sourced /home/ngirard/.local/share/sauce/soft.toml

LANG=C ls /home/ngirard/.local/share/sauce/soft.toml
ls: cannot access '/home/ngirard/.local/share/sauce/soft.toml': No such file or directory
```

I find it unexpected that sauce reports to have sourced a file that doesn't exist.

Great project btw, expect more feedback from me anytime soon here or on Reddit ;-)
Cheers

**Comment by DanCardin** [comment]
mhm. Fair point.

It's not 100% obvious to me what the correct behavior/messaging ought to be for calls to `sauce` when you're at `~/foo/bar/baz/` and there's only a saucefile available for ~/foo and/or ~/foo/bar. (which is probably the reason I switched from erroring, as that used to be the behavior).

It might make sense to switch to something more like:
```
Sourced sauce.toml, sauce/foo.toml, and sauce/foo/soft.toml from /home/ngirard/.local/share
```
although I can already imagine ways in which that might also be problematic. Definitely open to suggestions!

**Comment by DanCardin** [comment]
https://github.com/DanCardin/sauce/pull/32 does essentially this ^. Will be released as 0.6.6

**Comment by ngirard** [comment]
Hey Dan,
thanks for your efforts !
I didn't forget about your comments about my other issues ; I'll reply to them as soon as I can.
Cheers !
